src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -405,76 +408,86 @@ void FancyZones::ToggleEditor() noexcept
         m_terminateEditorEvent.reset(CreateEvent(nullptr, true, false, nullptr));
     }
 
-    HMONITOR monitor{};
+    //HMONITOR monitor{};

```

Comment:
Don't leave comments, if you don't need it, delete it.

Reply:
I've removed it! ✅